README.md

```diff
@@ -161,6 +161,34 @@ Parameters should be defined as follows:
   Note that after test case ID asterisk should be added. This is necessary due
   to the construction of the flag `-t` (or `--test`)
 
+#### Known issues
+
+When trying to run tests, you might face errors related to libraries not
+linking properly. You can try to fix that issue by doing:
+
+```bash
+source ${PWD}/venv/bin/activate
+```
+
+Another issue is tests returning:
+
+```bash
+[ ERROR ] Execution stopped by user.
+```
+
+without doing anything, or writing any log files. When encountering this error,
+it might be useful to run tests with this extra flag:
+
+```bash
+-v snipeit:no
```

Comment:
This could be documented as a variable, bot rather not in `known issues` section. We can document this variable in usage section.

Reply:
i moved info about `-v snipeit:no` [here](https://github.com/Dasharo/open-source-firmware-validation/pull/12/commits/263532fb3c2da8abbd4c47df686e810a2fb0920f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L189) and [here](https://github.com/Dasharo/open-source-firmware-validation/pull/12/commits/263532fb3c2da8abbd4c47df686e810a2fb0920f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R125)

Reply:
This is nice, thanks